biorbd_optim/plot.py

```diff
@@ -1,8 +1,9 @@
 import multiprocessing as mp
 import numpy as np
 import tkinter
+import casadi
 
-from scipy import interpolate
+from scipy import interpolate, integrate
 from matplotlib import pyplot as plt
```

Comment:
It seems that integration was removed from this file. Is this still necessary?

Reply:
no !